### PR TITLE
Update docs to include a pointer to how sync works today

### PR DESCRIPTION
**Pull request by justinsb**
Thanks to bentheelder for the tip!

### CODE REVIEW

File: registry.k8s.io/MANUALLY_SYNCING_BUCKETS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Manually syncing buckets
 
 **NOTE**: we are no longer manually syncing buckets, however this may be a useful
-fallback reference.
+fallback reference.  Syncs are now done by the `gcs-to-s3-sync` prow job.
```
Comment (ameukam) [inline registry.k8s.io/MANUALLY_SYNCING_BUCKETS.md:4]:
```suggestion
fallback reference.  Syncs are now done by the [gcs-to-s3-sync](https://testgrid.k8s.io/sig-k8s-infra-k8sio#gcs-to-s3-sync) prow job.
```

**Comment by justinsb** [comment]
Thanks @ameukam ... incorporated your suggestion and made you a coauthor :-)

**Comment by BobyMCbobs** [comment]
Thank you for adding this @justinsb!

**Pull request merged**